behat-tests/features/bootstrap/FeatureContext.php

```diff
@@ -400,12 +424,14 @@ public function assertLinkVisibleRegion($link, $region) {
   }
 
   /**
-   * @Then I should not see the link :link visible in the :region( region)
+   * @Then I should not visibly see the link :link in the :region( region)
```

Comment:
Gotcha - that's what I was curious about.  All for being standard across the contexts.  Thanks for cleaning this up.